api/src/main/java/org/pmiops/workbench/rdr/RdrMapper.java

```diff
@@ -87,15 +90,31 @@ default void addOtherPopulationDetails(DbWorkspace source, @MappingTarget RdrWor
   @Mapping(source = "dbUser.demographicSurvey.raceEnum", target = "race")
   @Mapping(source = "dbUser.demographicSurvey.lgbtqIdentity", target = "lgbtqIdentity")
   @Mapping(source = "dbUser.demographicSurvey.identifiesAsLgbtq", target = "identifiesAsLgbtq")
+  @Mapping(source = "accessTiers", target = "accessTierShortNames")
   RdrResearcher toRdrResearcher(
-      DbUser dbUser, @Nullable DbVerifiedInstitutionalAffiliation verifiedInstitutionalAffiliation);
+      DbUser dbUser,
+      List<DbAccessTier> accessTiers,
+      @Nullable DbVerifiedInstitutionalAffiliation verifiedInstitutionalAffiliation);
 
   @Mapping(source = "institution.shortName", target = "institutionShortName")
   @Mapping(source = "institution.displayName", target = "institutionDisplayName")
   @Mapping(ignore = true, target = "institutionalRole")
   RdrResearcherVerifiedInstitutionalAffiliation toRdrAffiliation(
       DbVerifiedInstitutionalAffiliation v);
 
+  default List<RdrAccessTier> toRdrAccessTiers(List<DbAccessTier> dbAccessTiers) {
+    if (dbAccessTiers.isEmpty()) {
+      return ImmutableList.of(RdrAccessTier.UNSET);
```

Comment:
Is this desired, or an empty list?  The ultimate decision for the ticket is unclear.

Reply:
Thanks - in the comments I actually specify it will be an empty list. That simplifies this code as well.